features/user_can_edit_articles.feature

```diff
@@ -4,10 +4,15 @@ Feature: Journalist can edit articles
   I would like to be able to edit and resubmit my articles
 
 Background:
-  Given we have the following articles
-    | headline                  | content     |
-    | The awesome article       | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. |
-  And user is signed in
+  Given the following users exist
+    | email                 | role        |
+    | journalist@test.com   | journalist  |
```

Comment:
If the mail character is a Journalist, then please rename this file to reflect that